awx/main/tests/unit/test_validators.py

```diff
@@ -42,16 +73,6 @@ def test_valid_locked_rsa_key():
     assert pem_objects[0]['key_enc']
 
 
-def test_invalid_rsa_key():
-    invalid_key = TEST_SSH_KEY_DATA.replace('-----END', '----END')
-    with pytest.raises(ValidationError):
```

Comment:
if keys begin and end with `-{4,}` then this negative test is incorrect and should have passed.